### ISSUE TITLE
"Wrong operation" tooltip

### ISSUE THREAD

**Issue by Stranger2015** [java-prolog-connectivity/logtalk-intellij#8]
Operator error is encountered when an operator's name is the special character(s) and has no spaces between the name and an operand starting with parenthesis:
`abc ;(X,Y)`
Workaround: adding space between the name and the operand will remedy the error.

**Comment by sergio-castro** [comment]
Good catch, thanks for reporting! This is definitely a bug, I will try to get this fixed.
